client/cityrepdata.cpp

```diff
@@ -726,63 +726,83 @@ static QString cr_entry_cma(const struct city *pcity, const void *data)
      FUNC_TAG(hstate_concise)},
+    {true, 15, 1, nullptr, N_("?cma:Governor"), N_("City: Governor"),
+     nullptr, FUNC_TAG(cma)},
+    {false, 4, 1, N_("?plague risk [short]:Pla"), N_("(%)"),
+     N_("City: Plague risk per turn"), nullptr, FUNC_TAG(plague_risk)},
 
-    {false, 2, 1, nullptr, N_("?Happy workers:H"), N_("Workers: Happy"),
+    // CITIZEN columns
+    {false, 2, 1, nullptr, N_("?Happy workers:H"), N_("Citizens: Happy"),
      nullptr, FUNC_TAG(happy)},
-    {false, 2, 1, nullptr, N_("?Content workers:C"), N_("Workers: Content"),
+    {false, 2, 1, nullptr, N_("?Content workers:C"), N_("Citizens: Content"),
      nullptr, FUNC_TAG(content)},
-    {false, 2, 1, nullptr, N_("?Unhappy workers:U"), N_("Workers: Unhappy"),
+    {false, 2, 1, nullptr, N_("?Unhappy workers:U"), N_("Citizens: Unhappy"),
      nullptr, FUNC_TAG(unhappy)},
-    {false, 2, 1, nullptr, N_("?Angry workers:A"), N_("Workers: Angry"),
+    {false, 2, 1, nullptr, N_("?Angry workers:A"), N_("Citizens: Angry"),
      nullptr, FUNC_TAG(angry)},
-    {false, 10, 1, N_("?city:Workers"),
+    {false, 10, 1, N_("?city:Citizens"),
      N_("?happy/content/unhappy/angry:H/C/U/A"),
-     N_("Workers: Happy, Content, Unhappy, Angry"), nullptr,
+     N_("Citizens: Happy, Content, Unhappy, Angry"), nullptr,
      FUNC_TAG(workers)},
 
-    {false, 8, 1, N_("Best"), N_("attack"), N_("Best attacking units"),
+    // UNIT columns
+    {false, 8, 1, N_("Best"), N_("attack"), N_("Units: Best attackers"),
      nullptr, FUNC_TAG(attack)},
-    {false, 8, 1, N_("Best"), N_("defense"), N_("Best defending units"),
+    {false, 8, 1, N_("Best"), N_("defense"), N_("Units: Best defenders"),
      nullptr, FUNC_TAG(defense)},
     {false, 2, 1, N_("Units"),
      // TRANS: Header "Number of units inside city"
-     N_("?Present (units):Here"), N_("Number of units present"), nullptr,
+     N_("?Present (units):Here"), N_("Units: Number present"), nullptr,
      FUNC_TAG(present)},
     {false, 2, 1, N_("Units"),
      // TRANS: Header "Number of units supported by given city"
-     N_("?Supported (units):Owned"), N_("Number of units supported"),
-     nullptr, FUNC_TAG(supported)},
-
-    {// TRANS: Header "It will take this many turns before city grows"
-     true, 14, 1, N_("?food (population):Grow"),
-     N_("?Stock/Target:(Have/Need)"), N_("Turns until growth/famine"),
-     nullptr, FUNC_TAG(growturns)},
-
+     N_("?Supported (units):Owned"), N_("Units: Number supported"), nullptr,
+     FUNC_TAG(supported)},
+    // TRANS: "UpT" = "Units per turn"
+    {false, 3, 1, nullptr, N_("UpT"),
+     N_("Units: Maximum buildable per turn"), nullptr,
+     FUNC_TAG(build_slots)},
+
+    // RESOURCE columns
     {false, 10, 1, N_("Surplus"), N_("?food/production/trade:F/P/T"),
-     N_("Surplus: Food, Production, Trade"), nullptr, FUNC_TAG(resources)},
+     N_("Resources: Food, Production, Trade"), nullptr, FUNC_TAG(resources)},
```

Comment:
Either keep "surplus" in the name or go with "net" since the value is the net food, net prod, etc.

Also, Luxury or Luxuries needs to be Luxury Goods. We changed that awhile back. Not included in this PR, but a good time to change it with the other updates.